src/app/organisations/postConfirmEditOrganisation.js

```diff
@@ -0,0 +1,37 @@
+const { editOrganisation } = require("../../infrastructure/organisations");
+const logger = require("../../infrastructure/logger");
+
+const {
+  getOrganisationByIdV2,
+} = require("./../../infrastructure/organisations");
+
+const postConfirmEditOrganisation = async (req, res) => {
+  const correlationId = req.id;
+  const organisation = await getOrganisationByIdV2(req.params.id, req.id);
```

Comment:
2 minor changes.
- Can you do a check on the organisation we just got for the category to make sure it's 008.  It's super unlikely, but it stops someone messing around with the URL by hand to modify an org that really shouldn't be tinkered with.
- Can you do the session.formData check first, before getting the organisation?  If there isn't any session data then there's no point in doing the API call to get the organisation